README.md

```diff
@@ -1,70 +1,91 @@
-# Getting Started with Create React App
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/maosan132/calculator/">
+    <img src="./mLogo.png" alt="Logo" width="80" height="80">
+  </a>
```

Comment:
Thank you for making some of the required changes.
- [ ] As mentioned in the last review, please let's fix the live version link. It's currently displaying your README and different from what is displayed when the local version of your app is run.

Reply:
I don't get it. Please elaborate more. Do you mean changing the link on the logo of MIcroverse?

Reply:
No, I didn't mean that you should change the logo, you had a wrong link that was leading to another resource.